src/Transform/ONNX/ShapeInferencePass.cpp

```diff
@@ -53,6 +59,45 @@ class ShapeInferencePass
       }
     });
 
+    // Populate a set to keep track of all shape fold nodes
+    std::set<Operation *> shapeFoldSet;
+    f.walk([&](mlir::Operation *op) {
+      if (op->getAttr("shape_folding_value")
```

Comment:
Minor -- reuse `getShapeFoldingAttr`, ideally the string literal `shape_folding_value` should appear as few times as possible.